features/0023-did-exchange/README.md

```diff
@@ -325,6 +325,19 @@ The exchange response message is used to complete the exchange. This message is
             "signature": "3dZWsuru7QAVFUCtTd0s7uc1peYEijx4eyt5... (bytes omitted)"
             }
       }
+   },
+   "did_rotate~attach": {
```

Comment:
Two questions to consider — more to know why you did it this way then to propose changes.

- Should this be a change to the protocol version to 1.1?
   - It is a new item, so I wanted to check to see if you think there should be a version bump.
- Why the “_” and “-“ in the same item?
  - I see it in the did_doc-attach, so assume that is the reason, but thought I would ask :-).

Reply:
second item: it's a tilde, not a dash.
first item - maybe/probably? A topic for tomorrow.

Reply:
Doh!  Sorry about that — it flashed through my head that the attach was a decorator, but didn’t stick, and not when I was looking at the item name :-).

That makes it easier for this to be a no version change.  The guidance is to put a decorator on it with this content vs. the must.  Of course, any implementation can do what they think is appropriate — either including the `~attach` decorator or not, ignoring or processing the decorator on receipt, and rejecting or not the connection if NOT included.